backend/docker-compose.yml

```diff
@@ -62,6 +62,8 @@ services:
       MARIADB_USER: user
       MARIADB_PASSWORD: secret
       MARIADB_RANDOM_ROOT_PASSWORD: "true"
+    tmpfs:
```

Comment:
ça your elaborate what is the usage of tmpfs here ?

Reply:
It speeds up the tests a little as the tmpfs is in RAM and we obviously do not care that the test database is lost. It was already present in `.travis.yml`.